src/Cake.Core.Tests/Unit/CakeEngineTests.cs

```diff
@@ -977,6 +977,42 @@ public sealed class WithExecutionStrategy
                 Assert.Contains(fixture.Log.Entries, x => x.Message.StartsWith("Task Teardown error (A):"));
             }
 
+            [Fact]
+            public async Task Should_Return_Report_That_Contains_Entry_For_Setup_First()
+            {
+                // Given
+                var fixture = new CakeEngineFixture();
+                var engine = fixture.CreateEngine();
+                engine.RegisterSetupAction(x => { });
+                engine.RegisterTask("A");
+
+                // When
+                var report = await engine.RunTargetAsync(fixture.Context, fixture.ExecutionStrategy, "A");
+
+                // Then
+                Assert.Equal(2, report.Count());
+                Assert.Equal(Constants.Reporting.SetupEntry, report.ElementAt(0).TaskName);
```

Comment:
Can we change this to `Assert.Equal("**Setup**", report.ElementAt(0).TaskName);`? Makes it easier to read.